catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpecTest.groovy

```diff
@@ -691,4 +709,29 @@ class ValidationParserSpecTest extends Specification {
 }
 '''
 
+    String matchOneAttributeValidator = '''
+{
+    "validators": {
```

Comment:
I am not sure how to read this... `validators` has a `title` that is an array of validators?